site/docs/components/static-list/accessibility.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+## Keyboard interactions
+
+<KeyboardControls>
+<KeyboardControl keyOrCombos="Tab">
+
+- If focus is above the Static List, Tab key moves focus into the first interactive element in the List
+
+</KeyboardControl>
+<KeyboardControl keyOrCombos="Shift + Tab">
+
+- If focus is below the List, Tab key moves focus into the last interactive element in the static list; or the first interactive element in the Static List
+
+</KeyboardControl>
+<KeyboardControl keyOrCombos="Space / Enter">
+
+- Enter or Space keys press, will trigger the action of the selected interactive element.
```

Comment:
This isn't applicable to static list?

Reply:
@joshwooding can you expand your question? I don't fully understand it. Asmita [reviewed this](https://www.figma.com/design/pXXcdSsLVdsSuFgpnfj9L5/Static-list?node-id=9261-33127&t=0p16WLhg8GHo540w-1) with Jake and received the green light. LMK if we need to correct this, NP.

Reply:
What does "selected interactive element" mean? List is static and therefore isn't interactive

Reply:
Understood!